website/www/site/content/en/documentation/ml/overview.md

```diff
@@ -105,3 +105,4 @@ You can find examples of end-to-end AI/ML pipelines for several use cases:
 * [Anomaly Detection in Beam](/documentation/ml/anomaly-detection): Demonstrates how to set up an anomaly detection pipeline that reads text from Pub/Sub in real time and then detects anomalies using a trained HDBSCAN clustering model with the RunInference API.
 * [Large Language Model Inference in Beam](/documentation/ml/large-language-modeling): Demonstrates a pipeline that uses RunInference to perform translation with the T5 language model which contains 11 billion parameters.
 * [Per Entity Training in Beam](/documentation/ml/per-entity-training): Demonstrates a pipeline that trains a Decision Tree Classifier per education level for predicting if the salary of a person is >= 50k.
+* [Cross-language RunInference in Beam](/documentation/ml/multi-language-inference): Demonstrates a cross-language java pipeline that uses a custom python transform to do preprocessing, postprocessing, and inference.
```

Comment:
Note that this page is being changed in https://github.com/apache/beam/pull/25700/files. If that PR is merged first, this could be added either to the Data processing or to the Prediction and inference examples, depending on which you think is more appropriate.